pom.xml

```diff
@@ -29,6 +29,14 @@
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>com.vaadin.external.google</groupId>
+            <artifactId>android-json</artifactId>
```

Comment:
why did you choose this library? It seems odd to have android library for web app. I'm quite sure it would be enough to provide [gson Google library](https://github.com/google/gson).